week03_model_free/homework.ipynb

```diff
@@ -758,6 +758,59 @@
     "Also, don't forget to optimize ```learning_rate```, ```discount``` and ```epsilon``` params of model, this may also help to solve this env."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# The following commands install the environment \"Pacman\" requirements\n",
+    "# in Collab, the course docker container, Ubuntu and other OS that use the package manager \"APT-GET\"\n",
+    "#!apt-get update\n",
+    "#!apt-get install -y python-tk"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "#### Cell of the remote display preparation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "prepare_display_server = False  # set to True if you want visualization on the system without a physical display\n",
+    "\n",
+    "# The display_server_preparation.sh uses the package manager \"APT-GET\",\n",
+    "# if your OS uses another package manager you should CHANGE the script.\n",
+    "\n",
+    "if prepare_display_server: \n",
+    "    ! ./display_server_preparation.sh\n",
```

Comment:
By this point I'm not sure why one would prefer to introduce an extra boolean variable to control this instead of just commenting a line.

```python
# If you are running in an Ubuntu-based Docker image, uncomment this to enable visualization support.
# Otherwise, if you are running this notebook on a headless server, you may have some luck
# getting visualization to work by tweaking this script.

# ! ./display_server_preparation.sh
```

Reply:
Let it be some small self-documented piece of code.